test/router-luatest/reload_test.lua

```diff
@@ -0,0 +1,265 @@
+local t = require('luatest')
+local vtest = require('test.luatest_helpers.vtest')
+local git_util = require('test.lua_libs.git_util')
+local fio = require('fio')
+
+local g = t.group('reload_router')
+local cfg_template = {
+    sharding = {
+        {
+            replicas = {
+                replica_1_a = {
+                    master = true,
+                },
+                replica_1_b = {},
+            },
+        },
+        {
+            replicas = {
+                replica_2_a = {
+                    master = true,
+                },
+            },
+        },
+    },
+    bucket_count = 100
+}
+
+local global_cfg
+local latest_hash
+local vshard_copy_path
```

Comment:
I don't know are there any guidelines but cartridge tests save such variables directly to `g` (e.g. https://github.com/tarantool/cartridge/blob/5569997bac5ad74c792442d914d3e6d20c3df604/test/integration/restart_replication_test.lua#L20). Up to you.